src/Components/RideProgressCard/Components/ProgressLine.tsx

```diff
@@ -0,0 +1,82 @@
+import * as React from "react";
```

Comment:
Every component you create should be a module, which is a folder with index file exporting the default export.

This is a requirement, because you can create a much more flexible file structure for every module you have: e.g. separate styles files, add storybook, add tests, add README and etc.